recipes/leftbar/callbox/callbox.stories.js

```diff
@@ -44,6 +45,12 @@ const DefaultTemplate = (args) => createTemplateFromVueFile(
   DtRecipeCallboxDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
```

Comment:
Seems like it is not working cos this `argTypes`, remove it and it will works.

Reply:
Thanks for catching that! forgot to remove it while migrating to vue3